geopandas/tools/tests/test_sjoin.py

```diff
@@ -36,27 +36,26 @@ def dfs(request):
 
     df1 = GeoDataFrame({"geometry": polys1, "df1": [0, 1, 2]})
     df2 = GeoDataFrame({"geometry": polys2, "df2": [3, 4, 5]})
+
     if request.param == "string-index":
         df1.index = ["a", "b", "c"]
         df2.index = ["d", "e", "f"]
 
+    df1.index.name = "index_left"
+    df2.index.name = "index_right"
```

Comment:
Maybe use something slight different? as those match the automatically generated names based on the suffix (in case they become columns), to ensure we certainly use the original one and not those automatically generated name